modules/client-configuration/pages/supported-features.adoc

```diff
@@ -80,11 +80,11 @@ ifeval::[{suma-content} == true]
 | {x86_64}
 | {check}
 
-| {almalinux} 8
+| {almalinux} 9
```

Comment:
This not adjusting the architectures, and in fact is changing the usual order, as now 9 comes before 8? I don't think it's needed? Other OS go from older to newer